mantle/kola/tests/util/luks.go

```diff
@@ -79,3 +80,26 @@ func LUKSSanityTest(c cluster.TestCluster, tangd TangServer, m platform.Machine,
 	luksDump = c.MustSSH(m, "sudo cryptsetup luksDump "+rootPart)
 	mustMatch(c, "Cipher: *aes", luksDump)
 }
+
+// LUKSSanityCEXTest verifies that the rootfs is encrypted with Cex based LUKS
+func LUKSSanityCEXTest(c cluster.TestCluster, m platform.Machine, rootPart string) {
+	var err error
+	luksDump := c.MustSSH(m, "sudo cryptsetup luksDump "+rootPart)
+	mustMatch(c, "cipher: paes-*", luksDump)
+	mustNotMatch(c, "Cipher: *cipher_null-ecb", luksDump)
+	mustMatch(c, "0: paes-verification-pattern", luksDump)
+	mustNotMatch(c, "9: *coreos", luksDump)
+
+	err = m.Reboot()
+
+	if err != nil {
+		c.Fatalf("Failed to reboot the machine: %v", err)
+	}
+	luksDump = c.MustSSH(m, "sudo cryptsetup luksDump "+rootPart)
+	mustMatch(c, "cipher: paes-*", luksDump)
+
+	err = coretest.TestRHCOSGrowfs()
```

Comment:
OK, so the actual problem with this is that `coretest.TestRHCOSGrowfs()` assumes that it's running on the VM itself under test. In the basic tests, it's a native function. Here we're running it from the host so it's actually e.g. calling journalctl on your host system which is of course not what we want.

Will add comments for how we can fix this.